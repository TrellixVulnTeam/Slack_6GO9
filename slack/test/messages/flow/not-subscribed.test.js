const { NotSubscribed } = require('../../../lib/messages/flow');

describe('NotSubscribed message rendering', () => {
  test('works', async () => {
    const message = new NotSubscribed('atom/atom');
    expect(message.toJSON()).toMatchSnapshot();
  });
});
