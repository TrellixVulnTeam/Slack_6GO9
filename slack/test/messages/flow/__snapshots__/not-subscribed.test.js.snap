// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotSubscribed message rendering works 1`] = `
Object {
  "attachments": Array [
    Object {
      "color": "danger",
      "mrkdwn_in": Array [
        "text",
      ],
      "text": "You're not currently subscribed to \`atom/atom\`
Use \`/github subscribe atom/atom\` to subscribe.",
    },
  ],
  "response_type": "ephemeral",
}
`;
